python/instrumentation/openinference-instrumentation-agno/src/openinference/instrumentation/agno/_wrappers.py

```diff
@@ -270,21 +287,32 @@ def run_stream(
 
             try:
                 yield from wrapped(*args, **kwargs)
-                # Use get_last_run_output instead of removed agent.run_response
-                session_id = None
-                try:
-                    session_id = arguments.get("session_id")
-                except Exception:
-                    session_id = None
-
-                run_response = None
-                if hasattr(agent, "get_last_run_output"):
-                    run_response = agent.get_last_run_output(session_id=session_id)
+                if "session" in arguments and arguments.get("session") and len(arguments.get("session").runs) > 0:
+                    for run in arguments.get("session").runs:
+                        if run.content:
```

Comment:
But if the session has many runs, this won't work? You'll have to get the last run in the session.